.travis.yml

```diff
@@ -30,20 +30,14 @@ after_success:
   - ./gradlew jacocoTestReport coveralls
   - export IMAGE_TAG=faf-java-api;
   - export REPO=faforever/faf-java-api;
-  - if [ -n "${TRAVIS_TAG}" ]; then
+  - if [ -n "${TRAVIS_TAG}" || "${TRAVIS_BRANCH}" == "develop" ]; then
     docker build -t ${IMAGE_TAG} . &&
     echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin &&
     docker tag ${IMAGE_TAG} ${REPO}:${VERSION} &&
     docker push ${REPO};
     fi
 # TODO Codacy coverage reporter does not yet support Java 9. See https://github.com/codacy/codacy-coverage-reporter/issues/76
 #      ./gradlew sendCoverageToCodacy;
-  - if [ "${TRAVIS_BRANCH}" == "develop" ]; then
-    docker build -t ${IMAGE_TAG} . &&
-    echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin &&
-    docker tag ${IMAGE_TAG} ${REPO}:${VERSION} &&
-    docker push ${REPO};
-    fi
```

Comment:
I know it's not obvious but the reason why the block is duplicated is because for `develop`, `./gradlew sendCoverageToCodacy;` should be executed (which is currently commented out), but not for tags.
Since codacy/codacy-coverage-reporter#76 is solved, it can be put back into the `if`. If you see a good solution how to remove the redundancy, please feel free to do so :)

Reply:
What is the advantag of not sending to coverage on tags? Yes it's redundant, but it's not causing any troubles.

Reply:
Hmm I wouldn't know... maybe I thought it's only relevant (or even working) on `develop` but coveralls supports multibranch (other than SonarQube Community Edition) so we're good 👍